examples/resources/spectrocloud_virtual_machine/resource.tf

```diff
@@ -1,51 +1,72 @@
+data "spectrocloud_cluster" "vm-enabled-cluster" {
+  name = "newyork"
+  context = "tenant"
+}
+
 resource "spectrocloud_virtual_machine" "tf-test-vm-default" {
-  cluster_uid = "6406b8ee7c865766fde23247"
+  cluster_uid = data.spectrocloud_cluster.vm-enabled-cluster.id //"6406b8ee7c865766fde23247"
   name = "tf-test-vm-default"
   namespace = "default"
-  cpu_cores = 2
-  memory = "3G"
+  cpu_cores = 1
+  memory = "2G"
   image_url = "quay.io/kubevirt/fedora-cloud-container-disk-demo"
-  vm_action = "migrate"
-  labels = ["tf-test=true", "env=dev"]
+  labels = ["tf-test=true", "env=sumit-dev"]
   annotations = {
     tf : "true",
-    owner: "siva",
+    owner: "siva1",
   }
 }
 
+resource "spectrocloud_virtual_machine" "tf-test-vm-clone-default" {
+  cluster_uid = data.spectrocloud_cluster.vm-enabled-cluster.id
+  clone_name = spectrocloud_virtual_machine.tf-test-vm-default.name
+  name = "tf-test-vm-clone-default"
+  namespace = "default"
+  depends_on = [spectrocloud_virtual_machine.tf-test-vm-default]
```

Comment:
shouldn't be necessary. as line 22 already have implicit dependency.

Reply:
Removed the explicit dependency